### PR TITLE
Improve naming

### PR DESCRIPTION
**Pull request by sealor**

### CODE REVIEW

File: junkie/context.py [hunk 1]
```diff
@@ -19,18 +19,18 @@ def add(self, *singletons_and_factories_args: Union[Dict[str, object], Dict[str,
                     self._singletons[key] = value
 
     @contextmanager
-    def build(self, instances: Union[Set[str], List[str], type, str]):
-        if isinstance(instances, set) or isinstance(instances, list):
-            with self.build_dict(instances) as instance_dict:
+    def build(self, names_or_type: Union[Set[str], List[str], str, type]):
```
Comment (Rollmops) [inline junkie/context.py]: `names_or_types`?

File: junkie/context.py [hunk 2]
```diff
@@ -19,18 +19,18 @@ def add(self, *singletons_and_factories_args: Union[Dict[str, object], Dict[str,
...
+        if isinstance(names_or_type, set) or isinstance(names_or_type, list):
```
Comment (Rollmops) [inline junkie/context.py]: `isinstance(names_or_type, (set, list))`

File: junkie/context.py (hunk 1 above)
Comment (Rollmops) [reply junkie/context.py]: Oder geht da nur ein type Argument?

Comment (sealor) [reply junkie/context.py]: Indeed, currently you can only have a single type here. All sets and lists expect strings.
Do you think both sould be supported (str and type in lists and sets)?
PR? ;-)

File: junkie/context.py (hunk 2 above)
Comment (sealor) [reply junkie/context.py]: Thx!

**Pull request merged**